python/ray/serve/scripts.py

```diff
@@ -23,6 +22,16 @@
 from ray.serve.application import Application
 from ray.serve.api import Deployment
 
+RAY_INIT_ADDRESS_HELP_STR = (
+    "Address to use for ray.init(). Can also be specified "
+    "using the RAY_ADDRESS environment variable."
+)
+RAY_DASHBOARD_ADDRESS_HELP_STR = (
+    "Address to use to query the Ray dashboard (defaults to "
+    "http://localhost:8265). Can also be specified using the "
+    "RAY_ADDRESS environment variable."
```

Comment:
I also wasn't sure about this. `RAY_ADDRESS` is normally the address + port of the head node, correct? But the Ray dashboard isn't at that same address. Should we ever default to the `RAY_ADDRESS` for the dashboard?

Reply:
resolved offline